Source/Scene/CameraEventAggregator.js

```diff
@@ -262,7 +262,8 @@ define([
         }
         //>>includeEnd('debug');
 
-        this._eventHandler = new ScreenSpaceEventHandler(canvas);
+        this._eventHandler = new ScreenSpaceEventHandler(canvas, false);
```

Comment:
I see the warning when I open Viewer though. I think I missed the argument somewhere.